sktime/forecasting/base/__init__.py

```diff
@@ -8,5 +8,5 @@
     "BaseForecaster",
 ]
 
-from sktime.forecasting.base._fh import ForecastingHorizon
 from sktime.forecasting.base._base import BaseForecaster
+from sktime.forecasting.base._fh import ForecastingHorizon
```

Comment:
Is `Install and test / run-notebook-examples (pull_request)` and `Install and test / test-nosoftdeps (pull_request)` failing due to this moving of imports?

Reply:
yes, it seems somehow it is causing a circular import - the problem is not the linting change, the problem is that there is probably a bug which was masked by the wrong sequence in the first place.

Reply:
attempt to resolve this:
https://github.com/sktime/sktime/pull/4329

seems to happen when we start at `conftest` - alternative hypothesis, any import of `test_all_estimators` will cause this

Reply:
correction: it was the forecaster base module, it was importing from base, but was itself imported from base